<launch>
    <include file = "$(find localization)/launch/racecar_model.launch"/>


    <include file = "$(find zed_wrapper)/launch/zed.launch"/>

    <node pkg="final_challenge" type="pure_pursuit.py" name="Trajectory_follower">
		<param name="odom_topic" value="/pf/pose/odom"/>
		<param name="drive_topic" value="/vesc/high_level/ackermann_cmd_mux/input/nav_3"/>
		<param name="speed" value="0.5"/>
		<param name="lookahead" value="1.0"/>
	</node>

	<node pkg="final_challenge" type="orange_line_detector.py" name="Trajectory_planner" output="screen">
		<param name="image_topic" value="/zed/zed_node/rgb/image_rect_color"/>
		<param name="active_state" value="/line_follower"/>
		<param name="start_active" value="True"/> 
	</node>

	<node pkg="wall_follower" type="safety_controller.py" name="safety_controller">
	</node>
</launch>
